### ISSUE TITLE
Jinja card doesn't appear on the list of custom cards in HA

### ISSUE THREAD

**Issue by gcoan** [PiotrMachowski/Home-Assistant-Lovelace-HTML-Jinja2-Template-card#27]
Added HTML Jinja2 xcard successfully to HACS
![image](https://github.com/PiotrMachowski/Home-Assistant-Lovelace-HTML-Jinja2-Template-card/assets/142018870/596b343f-d32b-47da-933e-5a6d698bbaf6)

And it works fine if I manually add the card configuration to a Lovelace dashboard, but the card does not appear in the list of cards when I click on 'Add card' in HA:
![image](https://github.com/PiotrMachowski/Home-Assistant-Lovelace-HTML-Jinja2-Template-card/assets/142018870/5f4db94d-a01d-4c71-bb49-0bc1aab72b56)

HA 2024.1.5

**Comment by perlchamp** [comment]
it does:
in HACS searching for 

> HTML Jinja2 Template card

and install it.
Issue can be closed !

**Comment by gcoan** [comment]
> it does: in HACS searching for
> 
> > HTML Jinja2 Template card
> 
> and install it. Issue can be closed !

No, it doesn't!  Please read the enhancement description, the issue is not whether the card appears in HACS, it is whether it appears in the list of cards when editing a dashboard.

I just checked and it still doesn't

**Comment by perlchamp** [comment]
ah, ok, now i understand the problem. Sorry for wasting your time.
But i think the most custom-cards (i installed) aren't there (p.ex. browser-mod, card-mod, floorplan, ), are they?
So...

**Comment by perlchamp** [comment]
layout-card (and its friends), apexchart and the custom-button-card, too. Maybe some additional json-packages-stuff is missing for registering (for updating automatically and so on). I don't know. At the end they have to become configured via yaml. That's the cause why i don't miss this behavior (being listed under custom-cards)... for me the functionality is more important ...

**Comment by ibrewster** [comment]
Pardon my ignorance, but how are you supposed to use it if it doesn't appear to be added to a dashboard?

**Comment by PiotrMachowski** [comment]
@ibrewster you just have to open YAML editor and configure it manually (it is at the bottom of the list)

**Comment by gcoan** [comment]
Any chance this could be looked at?

I know its only minor, but HA now displays a list of custom cards when you are adding a card to the dashboard

![image](https://github.com/user-attachments/assets/8d490356-1d8d-4012-94e2-ed65a25a0395)

But Jinja card doesn't appear

![image](https://github.com/user-attachments/assets/7ebc3a6c-113c-474b-a93e-fd9ecb02c136)

Yes I know you can scroll to the bottom and select 'manual', but then you need to remember the precise card name to start filling in the YAML.

I suspect there is something missing from the card config that means it doesn't appear in the HA card list?
